app/jobs/segment_identify_job.rb

```diff
@@ -6,8 +8,13 @@ def perform(membership_id:)
     return if membership_id.nil? || membership_id == 'membership/unidentifiable'
 
     membership = Membership.find(membership_id.sub(/\Amembership\//, ''))
-    traits = { created_at: membership.created_at, hosting_type: hosting_type, version: version }
-    traits.merge!(email: membership.user.email) if hosting_type == 'cloud'
+    traits = {
+      created_at: membership.created_at,
+      hosting_type: hosting_type,
+      version: version,
+      organization_name: membership.organization.name,
```

Comment:
Are you sure we can track the organization's name on the self-hosted version as well? 🤔 

For instance, the email of the membership is only tracked on the cloud version.